packages/CollabCodeEdit-Server.package/CCEServer.class/class/start.st

```diff
@@ -1,5 +1,10 @@
 startup
 start
```

Comment:
Maybe a more wordy method name instead of a comment.

Reply:
Good suggestion. For the time being, I would, however, like to keep it like as is.
The naming was a conscious decision made in an effort to stay consistent with the instances' interface.